docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+
+  db:
+    build:
+      context: .
+      dockerfile: Dockerfile.postgres
+    ports:
+      - 5432:5432

```

Comment:
Bara ett förslag som sagt, inte säkert det är nödvändigt att ändra så vi inte använder postgres default port. Det kommer som sagt att krocka (port aldready in use) om man som jag har en lokal postgres installerad på datorn.